### DISCUSSION TITLE
Triggering password reset

### DISCUSSION THREAD

**Discussion by ghost** [area17/twill#1230] [Q&A] [answered]
Hi again!

Is there any way to reset a password for a user other than going through the 'forgot password?' flow? I'm trying to make it so only a superadmin user can reset passwords for other users

Cheers!

**Comment by pboivin** [comment, marked as answer]
Hi @aarontilt, I haven't tested this but I believe the following is all that's needed to trigger a password reset email:
```
use Illuminate\Support\Facades\Password;

// ...

Password::broker()->sendResetLink(['email' => $user->email]);
```

I think you could wrap this in a custom command, something like `php artisan twill:reset-password`. You can use this to get you started: https://gist.github.com/pboivin/8029a00e0e21584bb10d20ca5a6a96f3

You can hide the "Forgot password" link from the login form by overriding this view into you project:
https://github.com/area17/twill/blob/2.x/views/auth/login.blade.php#L14

You can also completely override the actual routes, if you want to go the extra mile :)

Hope this can help!

**Reply by ghost** [reply]
Thank you very much!
